python/nx-cugraph/nx_cugraph/interface.py

```diff
@@ -67,7 +67,11 @@ def key(testpath):
         louvain_different = "Louvain may be different due to RNG"
         no_string_dtype = "string edge values not currently supported"
 
-        xfail = {}
+        xfail = {
+            key(
+                "test_eigenvector_centrality.py:TestEigenvectorCentrality.test_P3"
+            ): "Power iteration failed (don't know why; networkx converges in 8 steps)"
```

Comment:
I have no idea why PLC's eigenvector centrality would fail to converge for this graph. NetworkX converges in 8-ish steps.
The graph has three nodes with the following indices:
- `src_indices = [0, 1, 1, 2]`
- `dst_indices = [1, 0, 2, 1]`

In matrix form, this looks like this:
```
[[0 1 0]
 [1 0 1]
 [0 1 0]]
```

Reply:
I will raise a separate issue for this.